formats/polus-tiledtiff-converter-plugin/README.md

```diff
@@ -50,12 +53,45 @@ This plugin takes one input argument and one output argument:
 | `input`       | Input image collection        | Input  | Path    | true     |
 | `output`      | Output image collection       | Output | Path    | true     |
 
-## Run the plugin
+## Example Code
 
-### Run the Docker Container
-
-```bash
-docker run -v /path/to/data:/data labshare/polus-tiledtiff-converter-plugin:1.1.0 \
-  --input /data/input \
-  --output /data/output
+```Linux
```

Comment:
It might be a good idea to include a one sentence description of what this code does. Something like "Download an example dataset and convert data to tiled tiff using the tiled tiff converter plugin".

Reply:
Thank you for your suggestions. I have implemented the changes in my latest commit.